component-catalog/src/Examples/Button.elm

```diff
@@ -323,6 +330,28 @@ viewButtonExamples ellieLinkConfig state =
             , track = ShowItWorked "ButtonExample" "linkExternalWithTracking clicked"
             }
         ]
+    , form
+        [ css [ Css.marginTop Spacing.verticalSpacerPx ] ]
+        [ Button.button "Submit"
+            [ Button.submit
+            , Button.disabled
+            ]
+        ]
+    , div [ css [ Css.marginTop Spacing.verticalSpacerPx ] ]
```

Comment:
Could you add a heading describing this example? (Currently it's under "toggle button", which is inaccurate)